build.gradle

```diff
@@ -31,4 +31,6 @@ test {
     testLogging {
         events "passed", "skipped", "failed"
     }
-}
\ No newline at end of file
+}
+sourceCompatibility = JavaVersion.VERSION_14
+targetCompatibility = JavaVersion.VERSION_14
```

Comment:
We decided to use version 16, didn't we? also I don't think we need to specify the compatability.

Reply:
I had this issue where IntelliJ told me to add those lines, but if you use Java 16 as your JDK (rather than coretto-11) you shouldn't need those lines